api/v1/aerospikecluster_types.go

```diff
@@ -578,12 +589,19 @@ type AerospikeRoleSpec struct {
 
 // AerospikeUserSpec specifies an Aerospike database user, the secret name for the password and, associated roles.
 type AerospikeUserSpec struct {
+	// AerospikeAuthMode specifies an authentication mode (Internal or PKIOnly) enabled for the user.
+	// +kubebuilder:default:=Internal
```

Comment:
Just confirm whether this defaulting here will update the CR value, resulting in rolling restart for existing clusters when AKO is upgraded

Reply:
The change may start reflecting after any CR update performed post operator upgrade; however, this needs to be confirmed.

For my understanding: if we already have an enum validation and a default value set to Internal, what problem does omitempty solve in this case?